Sources/Instructions/Managers/Public/OverlayManager.swift

```diff
@@ -24,7 +24,7 @@ import UIKit
 
 // Overlay a blocking view on top of the screen and handle the cutout path
 // around the point of interest.
-public class OverlayManager {
+public class OverlayManager:NSObject {
```

Comment:
I don't see any valid reasons for OverlayManager to inherit NSObject. I can see that it was done in a80a085. From what I can understand from the commit message using automated translation, it was done to make it play well with swizzling in your project. While you certainly have good reasons for the change, I don't think it should be included in the base library.